src/main/java-templates/org/primefaces/component/slider/SliderTemplate.java

```diff
@@ -57,3 +60,60 @@ public void queueEvent(FacesEvent event) {
             super.queueEvent(event);
         }
     }
+
+    public final static String VALUE_OUT_OF_RANGE = "primefaces.slider.OUT_OF_RANGE";
+
+    @Override
+    public void validate(FacesContext context) {
+        super.validate(context);
+        
+        if(isValid()) {
```

Comment:
I prefer to use `if (!isValid()) { return; }` to short circuit the whole method rather than having all of the code inside a giant IF statement.  Just my preference.